kafka/integration_test.go

```diff
@@ -456,2169 +451,1945 @@ func consumerTest(t *testing.T, testname string, assignmentStrategy string, msgc
 
 }
 
-// Test consumer QueryWatermarkOffsets API
-func TestConsumerQueryWatermarkOffsets(t *testing.T) {
-	if !testconfRead() {
-		t.Skipf("Missing testconf.json")
+// use opaque string to locate the matching test message for message verification
+func findExpectedMessage(expected []*testmsgType, opaque string) *testmsgType {
```

Comment:
Just to confirm this:
The ordering in the file is now
(helper functions)
(integration suite struct definition)
(methods defined on the integration test suite struct)
(actual TestIntegration method which instantiates the suite)

Right?

Reply:
Yes it is that only